sound/soc/sof/nocodec.c

```diff
@@ -10,17 +10,20 @@
 
 #include <linux/module.h>
 #include <sound/sof.h>
+#include "sof-audio.h"
 #include "sof-priv.h"
 
 static struct snd_soc_card sof_nocodec_card = {
-	.name = "nocodec", /* the sof- prefix is added by the core */
+	.name = "sof-nocodec",
```

Comment:
Is the comment ``/* the sof- prefix is added by the core */`` invalid now?

Reply:
actually its partially valid. Its not needed but because of the way ignore_machine is set in the SOF driver, ignore_machine will take effect anyway and we might end up with the sof-sof-nocodec card name.

Reply:
Fixed now. I've left the card name as is.